### DISCUSSION TITLE
How to create a Project Template, for creating new projects based on another

### DISCUSSION THREAD

**Discussion by noahehall** [community/community#46355] [Projects and Issues] [answered]
### Select Topic Area

Product Feedback

### Body

I googled, but could not find how to create a project template, similar to repository templates.

I'm working on a [standard project definition for racexp](https://github.com/nirv-ai/racexp)
it would be great if I could create a template, and reuse it across my repositories

I am increasingly dreadful of the time it would take to bring [a project like this one](https://github.com/orgs/nirv-ai/projects/6) up to spec,
or to create new projects for other repositories.

If there is another way to go about doing this, that would be helpful as well

The core of it would be:
- default columns
- default fields
- default colum workflows
- modifying available labels

when creating a project, only the default templates are available
![image](https://user-images.githubusercontent.com/10324554/216776708-e4988b49-35bf-4e56-83d2-09dfdb9d00cf.png)

**Comment by noahehall** [comment]
looks like we can create a reusable github action thats triggered on [project create](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#project) 

from there we can configure the project using the github api

**Comment by rileybroughten** [comment, marked as answer]
Project templates are now in public beta! Check out the [changelog](https://github.blog/changelog/2023-05-25-github-issues-projects-may-25th-update/) announcement, and leave any thoughts or feedback on the [discussion post](https://github.com/orgs/community/discussions/54576) ✨ Project templates currently include any views, custom fields and draft issues (if you want them). Workflows are coming 👀

**Reply by noahehall** [reply]
that is awesome!
